hezar/models/image2text/crnn/crnn_image2text.py

```diff
@@ -53,6 +54,20 @@ def forward(self, pixel_values, **kwargs):
         x = nn.functional.log_softmax(x, 2)
         return x
 
+    def compute_loss(self, logits: torch.Tensor, labels: torch.Tensor, labels_lengths: torch.Tensor = None):
+        if labels_lengths is None:
```

Comment:
If labels_lengths should be None, why the default value is None?!

Reply:
@pooya-mohammadi Hmm, I don't remember exactly but I just tried to remove it and got warning (does not match signature ...) from the IDE.

Reply:
@pooya-mohammadi Done.